src/www/ui/api/documentation/openapi.yaml

```diff
@@ -2023,6 +2048,30 @@ components:
           type: string
           example:
             "Main group"
+    UserGroupMember:
+      type: object
+      properties:
+        user_pk:
+          type: integer
+          description: user's primary key
+          example: 1
+        user_name:
+          type: string
+          description: user name
+        user_status:
+          type: string
+          description: Status of the user
+        user_desc:
+          type: string
+          description: Description of the user
```

Comment:
Does it make sense to use the `#/components/schemas/User` object itself here?
https://github.com/fossology/fossology/blob/20ca3a482ce18e85859e92842e94fe06bb226237/src/www/ui/api/documentation/openapi.yaml#L1669-L1703